bdb/cursor.c

```diff
@@ -7686,6 +7686,12 @@ int bdb_osql_trak(char *sql, unsigned int *status)
         *status |= SQL_DBG_SHADOW;
         return 0;
     }
+
+    if (strncasecmp(sql, "OFF", 3) == 0) {
+        *status = 0;
+        return 0;
+    }
+
     return -1;
```

Comment:
I created a set directive ('set cursordebug <level>') which allows an sql session to enable our cursor-debug hooks in the cursor layer.  This allows someone to subsequently disable debug-trace by issuing "set cursordebug off".